psci/PSCi.hs

```diff
@@ -486,18 +486,29 @@ whenFileExists filePath f = do
     then f absPath
     else PSCI . outputStrLn $ "Couldn't locate: " ++ filePath
 
+-- |
+-- Attempts to read initial commands from '.psci' in the user's home then
+-- current directory
+--
 loadUserConfig :: IO (Maybe [Command])
 loadUserConfig = do
-  configFile <- (</> ".psci") <$> getCurrentDirectory
-  exists <- doesFileExist configFile
-  if exists
-  then do
-    ls <- lines <$> readFile configFile
+  localConfigPath <- configFileAt getCurrentDirectory
+  localExists <- doesFileExist localConfigPath
+
+  homeConfigPath <- configFileAt getHomeDirectory
+  homeExists <- doesFileExist homeConfigPath
+  case (localExists, homeExists) of
```

Comment:
Actually, `MaybeT` would probably do the job, since we don't need an error message.